### PR TITLE
Doc for SNMP topology

### PR DESCRIPTION
**Pull request by mpastorek**

### CODE REVIEW

File: frinx-uniconfig/user-guide/network-management-protocols/uniconfig-installing/readme.md [hunk 1]
```diff
@@ -638,3 +638,80 @@ curl --location --request POST 'http://localhost:8181/rests/operations/connectio
...
+
+### Identification of remote agent
+
+List of basic connection parameters that are used for identification of
+remote agent.
+
+- **node-id** - Name of node that represents device / mount-point in
+  the topology.
+- **snmp-topology:host** - IP address or domain-name of target
+  device that running an SNMP agent.
```
Comment (jaro0149) [inline frinx-uniconfig/user-guide/network-management-protocols/uniconfig-installing/readme.md:652]: device where SNMP agent is running

File: frinx-uniconfig/user-guide/network-management-protocols/uniconfig-installing/readme.md [hunk 2]
```diff
@@ -638,3 +638,80 @@ curl --location --request POST 'http://localhost:8181/rests/operations/connectio
...
+- **snmp-topology:request-timeout** - Timeout in milliseconds before 
+  a confirmed request is resent or timed out. Default value is 3000.
+- **snmp-topology:get-bulk-size** - The maximum number of values 
+  that can be returned in a single response to the get-bulk operation.
+  Default value is 50.
+
+### Authentication parameters
+
+- **snmp-topology:community-string** - UniConfig currently supports only
+  security string as authentication method that is connected with V1 and
```
Comment (jaro0149) [inline frinx-uniconfig/user-guide/network-management-protocols/uniconfig-installing/readme.md:673]: ... that is used with

File: frinx-uniconfig/user-guide/network-management-protocols/uniconfig_snmp/readme.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+Example of requests
+-------------------
+
+UniConfig currently supports read operation, with plans to add write operation in the future.
+
+## GET request
+
+```bash
+curl --location 'http://localhost:8181/rests/data/network-topology:network-topology/topology=topology-snmp/node=agent1/yang-ext:mount/SNMPv2-SMI:iso'
+```
```
Comment (jaro0149) [inline frinx-uniconfig/user-guide/network-management-protocols/uniconfig_snmp/readme.md:67]: you can add also sample response

**Review by jaro0149 — CHANGES_REQUESTED** [review]

File: frinx-uniconfig/user-guide/network-management-protocols/uniconfig-installing/readme.md (hunk 1 above)
Comment (mpastorek) [reply frinx-uniconfig/user-guide/network-management-protocols/uniconfig-installing/readme.md]: done

File: frinx-uniconfig/user-guide/network-management-protocols/uniconfig-installing/readme.md (hunk 2 above)
Comment (mpastorek) [reply frinx-uniconfig/user-guide/network-management-protocols/uniconfig-installing/readme.md]: done

File: frinx-uniconfig/user-guide/network-management-protocols/uniconfig_snmp/readme.md (hunk 3 above)
Comment (mpastorek) [reply frinx-uniconfig/user-guide/network-management-protocols/uniconfig_snmp/readme.md:67]: done

**Review by jaro0149 — APPROVED** [review]

**Pull request merged**